### PR TITLE
Update template for event hub

### PR DESCRIPTION
**Pull request by choldgraf**
This is a quick update to our hub template after filling it out once for OpenScapes and finding a few places where it'd help to have extra info.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 1]
```diff
@@ -21,22 +21,45 @@ assignees: ''
...
-  - Any `nbgitpuller` links resolve properly
-  - Notebooks run as-expected
+  - <details>
+    <summary>👉Template message to send to community representative</summary>
+    
+    ```
+    Hey {{ COMMUNITY REPRESENTATIVE }}, the date of your event is getting close!
+    
+    Could you please confirm that your hub environment is ready-to-go, and matches your hub's infrastructure setup, by ensuring the following things:
+       - [ ] Log-in and authentication works as-expected
```
Comment (choldgraf) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:36]:
```suggestion
     - [ ] Log-in and authentication works as-expected

```

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 2]
```diff
@@ -21,22 +21,45 @@ assignees: ''
...
+      
+     ```
+     Hey {{ COMMUNITY REPRESENTATIVE }}, your event appears to be over 🎉
+     
+     We hope that your hub worked out well for you! We are trying to understand where we can improve our hub infrastructure and setup around events, and would love any feedback that you're willing to give. Would you mind answering the following questions? If not, just let us know and that is no problem!
+  
+     - Did the infrastructure behave as expected?
+     - Anything that was confusing or could be improved?
+     - Any extra functionality you wish you would have had?
+     - Are you willing to share a story about how you used the hub?
```
Comment (choldgraf) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:61]:
```suggestion
     - Could you share a story about how you used the hub?

```

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 3]
```diff
@@ -21,22 +21,47 @@ assignees: ''
...
-  - Any `nbgitpuller` links resolve properly
-  - Notebooks run as-expected
+  - <details>
+    <summary>👉Template message to send to community representative</summary>
+    
+    ```
+    Hey {{ COMMUNITY REPRESENTATIVE }}, the date of your event is getting close!
+    
+    Could you please confirm that your hub environment is ready-to-go, and matches your hub's infrastructure setup, by ensuring the following things:
+     - [ ] Log-in and authentication works as-expected
```
Comment (damianavila) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:36]: Remove additional blank space here, maybe?

File: .github/ISSUE_TEMPLATE/5_event-hub.md [hunk 4]
```diff
@@ -21,22 +21,47 @@ assignees: ''
...
+
+    - [ ] `nbgitpuller` links you intend to use resolve properly
+    - [ ] Your notebooks run as-expected
+    
+    </details>
```
Comment (choldgraf) [inline .github/ISSUE_TEMPLATE/5_event-hub.md:41]:
```suggestion
    Could you please confirm that your hub environment is ready-to-go, and matches your hub's infrastructure setup, by ensuring the following things:
    - [ ] Log-in and authentication works as-expected
    - [ ] `nbgitpuller` links you intend to use resolve properly
    - [ ] Your notebooks run as-expected
    
    </details>  

```

**Comment by damianavila** [comment]
Opened a follow-up PR for something you missed from my review 😉 : https://github.com/2i2c-org/infrastructure/pull/852

**Pull request merged**